.github/workflows/axosyslog-docker.yml

```diff
@@ -149,13 +151,21 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: Extract metadata (AxoSyslog version) for Docker
-        id: docker-metadata-tags
-        uses: docker/metadata-action@v5
-        with:
-          images: ${{ needs.prepare.outputs.image-name }}
-          tags: type=match,pattern=axosyslog-([0-9].*),group=1
-          sep-tags: ','
+      - id: docker-metadata-tags
+        name: Extract metadata (AxoSyslog version) for Docker
+        run: |
+          last_full_version=$(skopeo inspect docker://ghcr.io/axoflow/axosyslog:latest | jq '.RepoTags' | sed 's/ \|"\|,//g' | grep "^[0-9]*.[0-9]*.[0-9]*$\|^[0-9]*.[0-9]*.[0-9]*-[0-9]*$" | tail -n 1)
```

Comment:
I think this won't work properly if we want to do a new AxoSyslog release: `4.9.0 -> 4.10.0`

When doing a stable release, this job is usually executed on a git tag.
I think it would be the safest and cleanest to extract the X.Y.Z version information always from a tag, and optionally, from an input parameter when we are forced to do a rebuild from a custom branch (that won't happen often).
`docker/metadata-action@v5` is a good tool to get such tag information.

When we want to calculate the container-rebuild-version `R` (`X.Y.Z-R`), we can do what you did: check the previous tag (but let's be strict: check if the last tag's X.Y.Z prefix matches the tag informtaion we extracted from  docker/metadata-action@v5 or from the input parameter), and increment `R` by one.

When doing a new AxoSyslog release `4.9.0 -> 4.10.0`, the X.Y.Z prefix won't match, in that case we should start from `1`: `4.10.0-1`.